lib/js/jsconsole.nim

```diff
@@ -110,5 +110,14 @@ since (1, 5):
     {.line: loc.}:
       {.emit: ["console.assert(", assertion, ", ", msg, ");"].}
 
+  func dir*(console: Console; obj: auto) {.importcpp.}
```

Comment:
```suggestion
  func dir*(console: Console; obj: auto) {.importjs.}
```
IIRC `{.importjs.}` does the same, but indicated intent better (ditto for the other cases).

Reply:
I agree, but the whole module is using `importcpp`, pre-existent code, 
maybe on a future PR all functions should be changed to `importjs`.

Reply:
Since your PR is called "improve jsconsole", I don't see a reason why you shouldn't do it in this PR. But feel free to defer that to a later PR.

Reply:
I agree, but I like to make tiny diff PR, so is easy to merge, easy to review, easy to backport, easy to revert. 
:)